### DISCUSSION TITLE
Display Campaigns from accounts related with Account Contact Relationship

### DISCUSSION THREAD

**Discussion by bagier26** [deejay-hub/timeline-lwc#298] [Q&A] [answered]
Would that be possible to display from the **Account** Time Warp, the **Campaigns associated with Contacts related to the Account ?**
I found very helpful to do the equivalent for task and events through  OpenActivity and ActivityHistory relationships, but not able to configure for Campaign Members.
Thank you !

**Comment by deejay-hub** [comment, marked as answer]
No.

Time Warp is primarily designed to plot child objects to the parent it is placed on. Activities for the Account object are the only exception.

You would have to fork the code base and add the code yourself to get access to records beyond that.
